doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -0,0 +1,117 @@
+[id='ai-credit-card-con_{context}']
+
+= Credit card fraud dispute use case
+
+The financial industry uses pragmatic AI for decisioning in several areas. One area is credit card charge disputes. When a customer identifies an incorrect or unrecognized charge on a credit card bill, the customer can dispute the charge.  Human intervention in credit card fraud detection is required in some cases but the majority of reported credit card fraud can be completely or partially resolved with pragmatic AI.
+
+This case study is about the fictitious Fortress Bank, the bank's customer Joe, and the business process management (BPM) developer Michelle. First we will look at how the bank originally used AI through {PRODUCT} digital decisioning and then we will see how Michelle enhanced the decision model with a Predictive Model Markup Language (PMML) model created from machine learning.
+
+Machine learning models such as Tensorflow and R produce predictive models. You can save these predicitive models in an open standard such as PMML so that you can use the model with {PRODUCT} or other products that support the PMML standard.
+
+Fortress Bank uses {PRODUCT} on {OPENSHIFT} to develop and run the Fortress Bank decision services. {OPENSHIFT} is a cloud platform for developing and running containerized applications. It is designed to enable applications and the data centers that support them to expand from just a few machines and applications to thousands of machines that serve millions of clients. {PRODUCT} is the Red Hat middleware platform for creating cloud-native business automation applications and microservices. It enables enterprise business and IT users to document, simulate, manage, automate, and monitor business processes and decisions. {CENTRAL} is the {PRODUCT} dashboard.
+
+== Credit card dispute digital decisioning with {PRODUCT}
+Joe is a Fortress Bank customer. Each month, he logs in to the Fortress Bank web site to review all of the charges on his bill before he pays it. This month, Joe sees a transaction that he recognizes but the amount is incorrect. The vendor has charged him 77 dollars instead of 47 dollars. Joe selects the row that contains the incorrect item and clicks *Dispute*.
```

Comment:
To make it consistent with the video/demo/screenshot I suggest as follows:

```suggestion
Joe is a Fortress Bank customer. Each month, he logs in to the Fortress Bank web site to review all of the charges on his bill before he pays it. This month, Joe sees a transaction that he recognizes but the amount is incorrect. The vendor has charged him 44.5 dollars instead of 4.5 dollars. Joe selects the row that contains the incorrect item and clicks *Dispute*.
```